test/test_orcid.py

```diff
@@ -0,0 +1,28 @@
+import pytest
+import responses
+from requests import HTTPError
+
+from profiles.orcid import OrcidClient
+
+
+@responses.activate
+def test_it_gets_a_record():
+    responses.add(responses.GET, 'http://www.example.com/api/v2.0/0000-0002-1825-0097/record',
+                  status=200, json={'foo': 'bar'})
```

Comment:
Doesn't look to be a way to check the rights headers were on the request (`Accept` and `Authorization`).

Reply:
https://pypi.python.org/pypi/requests-mock seems to allow matching on headers
Perhaps it's easier to keep it simple and `@patch('requests.get')`